app/models/chargeback.rb

```diff
@@ -170,4 +170,8 @@ def self.get_report_time_range(options, interval, tz)
   def self.report_cb_model(model)
     model.gsub(/^Chargeback/, "")
   end
+
+  def self.chargeback_model?(model)
+    %w(ChargebackContainerProject ChargebackVm).include?(model)
```

Comment:
@zeari enumerating the subclasses doesn't seem a good practice (not scalable/extensible in general). Would it be possible to use a `kind_of?` here?